.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,54 @@
+name: 💅 Feature design / RFC
+description: Submit a detailed feature request with a concrete proposal
+labels: [feature, triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Important things:
+
+        - We expect the feature request to be detailed.
+        - The request does not have to be perfect, we'll discuss it and fix it if needed.
+        - For a more "casual" feature request, consider using Canny instead: https://pact.canny.io.
```

Comment:
We may want to consider is the proposal is wider than just pact-python, it may relate to the pact-cores or the pact-specification in which case, pointing to those repos may be a better place.

but would love to see this applied across all the repos (or something similar which meets maintainers preferences) - I think this is solid though, as it makes you really consider stuff well.

Here was one for reference that I completed for docu https://github.com/facebook/docusaurus/issues/7008

Reply:
I think that answering this question is best placed with the maintainers, as opposed to the person asking. If they have a feature request which would impact the Pact Core, we can always move the issue to the appropriate repo as part of the triaging process. What do you think?

Reply:
>I think that answering this question is best placed with the maintainers, as opposed to the person asking. 

Agreed, they may have an inkling though - I suppose as long as they raise it somewhere, its gets triaged and moved to the _correct_ place as appropriate

>If they have a feature request which would impact the Pact Core, we can always move the issue to the appropriate repo as part of the triaging process.

you can transfer issues easily to different repos, and any tickets raised here will pop into the pact triage board, so that sounds sensible 👍🏾